.changeset/cyan-numbers-change.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sveltejs/adapter-cloudflare': patch
+'@sveltejs/adapter-node': patch
+---
+
+Pass `platform` object to SvelteKit
```

Comment:
Do we want to say in the changelog what it contains? If so, we're presumably want separate changesets for the Cloudflare and Node adapters.